lib/transaction/contracts.ts

```diff
@@ -0,0 +1,356 @@
+import utils from "../utils"
+import * as Contracts from './proto/contracts'
+import * as proto from "./../../google/protobuf/any";
+import { 
+  Transaction as T,
+  TXContract_ContractType,
+} from './proto/transaction'
+
+
+const enc = new TextDecoder();
+
+const decodeTransferContract = (wired: Uint8Array): any => {
```

Comment:
don't use type `any`, create the correct type

Reply:
transaction cant by typed...